src/styles/variables.json

```diff
@@ -96,7 +96,9 @@
     "overlay-z-index": "190",
     "tooltip-z-index": "190",
     "bdl-border-radius-size": "4px",
-    "bdl-border-radius-large-size": "10px",
```

Comment:
Is this value used anywhere today and, if so, do we need to update those usages?

Reply:
https://github.com/box/box-ui-elements/pull/2740/files#diff-fca7649e0a5992886ed74ba82a2ff242ad77703cbf9b3fd25a060f231e84837cR8

It was created for the use case in that file and used only there